### PR TITLE
Incorporating DNVGL buckling constraints for towers and monopiles

### PR DESCRIPTION
**Pull request by gbarter**
## Purpose
Jake did a great job coding up the DNV-GL buckling constraints.  This was driven by the need to switch from the older Eurocode standard for steel columns in buildings when it comes to floating platforms.  The implementation in FloatingSE is still pending, but this is now incorporated and the default for TowerSE

## Type of change
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
- [x] I have run existing tests which pass locally with my changes
- [ ] I have added new tests or examples that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: examples/05_tower_monopile/monopile_direct.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
 modeling_options["WISDEM"] = {}
 modeling_options["WISDEM"]["TowerSE"] = {}
 modeling_options["WISDEM"]["TowerSE"]["buckling_length"] = 30.0
+modeling_options["WISDEM"]["TowerSE"]["buckling_method"] = "euro"
```
Comment (ptrbortolotti) [inline examples/05_tower_monopile/monopile_direct.py:43]: should this be eurocode?

**Review by ptrbortolotti — APPROVED** [review]
nice job

File: examples/05_tower_monopile/monopile_direct.py (hunk 1 above)
Comment (gbarter) [reply examples/05_tower_monopile/monopile_direct.py]: Good catch!  Just fixed it.

**Review by JakeNunemaker — APPROVED** [review]

**Comment by JakeNunemaker** [comment]
I didn't realize this was waiting on me - looks good to me. Thanks for wrapping this into WISDEM for me @gbarter

**Pull request merged**